jaknaeso-server/src/main/java/org/nexters/jaknaesoserver/config/SecurityConfig.java

```diff
@@ -32,6 +34,7 @@ public SecurityFilterChain filterChain(HttpSecurity httpSecurity) throws Excepti
                     .permitAll()
                     .anyRequest()
                     .authenticated())
+        .cors(corsConfigurer -> corsConfigurer.configurationSource(corsConfigurationSource))
```

Comment:
이건 그냥 궁금해서 물어보는건데 제가 공식문서를 제대로 읽은건지 모르겠는데..기존 세팅에서

```suggestion
        .cors(withDefaults())
```
쓰면 저희 Mvc 설정 따라가는걸까요 ??

https://docs.spring.io/spring-security/reference/servlet/integrations/cors.html
해당문서참고했습니다 ! PR머지 후 확인해봐도 될것같습니다

Reply:
넵 감사합니다! 확인해볼게요!